wagtail_webstories/templates/wagtail_webstories/admin/import.html

```diff
@@ -27,7 +27,7 @@
                     {% if field.is_hidden %}
                         {{ field }}
                     {% else %}
-                        {% include "wagtailadmin/shared/field_as_li.html" with field=field %}
+                        {% include "wagtailadmin/shared/field.html" with field=field %}
```

Comment:
If we're switching from `field_as_li.html` to `field.html`, I think we need to provide our own `<li>` wrapper:
```suggestion
                        <li>{% include "wagtailadmin/shared/field.html" with field=field %}</li>
```